### PR TITLE
Enum updates to support Hourly reports

### PR DESCRIPTION
**Pull request by shalinved**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by nschonni** [comment]
Fixes #4666

**Review by sarangan12 — APPROVED** [review]

File: specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-12-01-preview/consumption.json [hunk 1]
```diff
@@ -0,0 +1,969 @@
+{
```
Comment (sarangan12) [inline specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-12-01-preview/consumption.json]: Where is this file used? This is not referenced in the readme file at all. Could you explain?

**Review by sarangan12 — CHANGES_REQUESTED** [review]

File: specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-12-01-preview/consumption.json (hunk 1 above)
Comment (sarangan12) [reply specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-12-01-preview/consumption.json]: @shalinved  Any response on this comment?

Comment (shalinved) [reply specification/consumption/resource-manager/Microsoft.Consumption/preview/2018-12-01-preview/consumption.json]: These files shouldn't have been included in this PR in the first place. They have nothing to do with this commit. I have removed these files from this PR for now.

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**